_posts/2021-07-05-newsletter.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: newsletter
+title: "Newsletter #11 - The Christmas Issue"
```

Comment:
hacker news is very strict about how submissions are titled, let's make sure Neovim is in the title.
```suggestion
title: "Neovim news #11 - The Christmas Issue"
```

Reply:
That'd be a "rebranding", but I like it (will insist on title case, though ;))